<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>katana</class>
 <widget class="QMainWindow" name="katana">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>609</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Purisa</family>
    <pointsize>12</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Katana Backup Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Purisa</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Backup name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="font">
         <font>
          <family>Purisa</family>
          <pointsize>14</pointsize>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="font">
         <font>
          <family>Purisa</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Folder to Backup</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../build-Katana-Qt_5_15_3_qt5-Debug/resources/folder.png</normaloff>../build-Katana-Qt_5_15_3_qt5-Debug/resources/folder.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="maximumSize">
         <size>
          <width>850</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Purisa</family>
          <pointsize>10</pointsize>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Purisa</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Compression type:      </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="font">
         <font>
          <family>Purisa</family>
          <pointsize>14</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>tar.gz</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../build-Katana-Qt_5_15_3_qt5-Debug/resources/targz.png</normaloff>../build-Katana-Qt_5_15_3_qt5-Debug/resources/targz.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_3">
        <property name="font">
         <font>
          <family>Purisa</family>
          <pointsize>14</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>zip</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../build-Katana-Qt_5_15_3_qt5-Debug/resources/zip.png</normaloff>../build-Katana-Qt_5_15_3_qt5-Debug/resources/zip.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTextBrowser" name="textBrowser_2"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>14</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>48</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <family>Purisa</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Backup</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../build-Katana-Qt_5_15_3_qt5-Debug/resources/backup.png</normaloff>../build-Katana-Qt_5_15_3_qt5-Debug/resources/backup.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="font">
         <font>
          <family>Purisa</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>About</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../build-Katana-Qt_5_15_3_qt5-Debug/resources/about.png</normaloff>../build-Katana-Qt_5_15_3_qt5-Debug/resources/about.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="font">
         <font>
          <family>Purisa</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Quit</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../build-Katana-Qt_5_15_3_qt5-Debug/resources/quit.png</normaloff>../build-Katana-Qt_5_15_3_qt5-Debug/resources/quit.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>727</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="font">
     <font>
      <family>Purisa</family>
      <pointsize>14</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_folder"/>
    <addaction name="actionAbout"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>../build-Katana-Qt_5_15_3_qt5-Debug/resources/about.png</normaloff>../build-Katana-Qt_5_15_3_qt5-Debug/resources/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset>
     <normaloff>../build-Katana-Qt_5_15_3_qt5-Debug/resources/quit.png</normaloff>../build-Katana-Qt_5_15_3_qt5-Debug/resources/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen_folder">
   <property name="icon">
    <iconset>
     <normaloff>resources/folder.png</normaloff>resources/folder.png</iconset>
   </property>
   <property name="text">
    <string>Open folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
